### PR TITLE
chore: support for Jubilee

### PR DESCRIPTION
**Pull request by lgalabru**

### CODE REVIEW

File: components/chainhook-types-rs/src/rosetta.rs [hunk 1]
```diff
@@ -341,19 +341,24 @@ pub enum OrdinalInscriptionTransferDestination {
...
+    NotAtOffsetZero,
+    NotInFirstInput,
+    Pointer,
+    Pushnum,
     Reinscription,
-    Unknown,
+    Stutter,
+    UnrecognizedEvenField,
+    Generic,
 }
```
Comment (MicaiahReid) [inline components/chainhook-types-rs/src/rosetta.rs:354]: @lgalabru do you have any info on why these changes are necessary?

Comment (lgalabru) [reply components/chainhook-types-rs/src/rosetta.rs:354]: Yeah, we're just aligning our list of curses with the one from `ord`.

**Review by MicaiahReid — APPROVED** [review]
